static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -546,7 +545,7 @@ const spanOperationBreakdownRenderer = (field: string) => (
           showDetail={false}
           spanBarHatch={false}
         >
-          {getHumanDuration(spanOpDuration / 1000)}
+          <Duration seconds={spanOpDuration / 1000} fixedDigits={2} abbreviation />
```

Comment:
You can also use `PerformanceDuration` and pass it `milliseconds={spanOpDuration}`


https://github.com/getsentry/sentry/blob/master/static/app/views/performance/utils.tsx#L170